src/VatRates/TaxesEuropeDatabaseClient.php

```diff
@@ -0,0 +1,93 @@
+<?php declare(strict_types = 1);
+
+namespace SandwaveIo\Vat\VatRates;
+
+use DateTime;
+use SoapClient;
+use SoapFault;
+
+final class TaxesEuropeDatabaseClient implements ResolvesVatRates
+{
+    const WSDL = 'https://ec.europa.eu/taxation_customs/tedb/ws/VatRetrievalService.wsdl';
+    const SOAP_CONFIG = [
+        'version' => SOAP_1_1,
+        'cache_wsdl' => WSDL_CACHE_MEMORY,
+    ];
+    /* @see https://ec.europa.eu/taxation_customs/tedb/ws/VatRetrievalServiceType.xsd */
+    const RATE_TYPE_STANDARD = 'STANDARD';
+    const RATE_VALUE_TYPE_DEFAULT = 'DEFAULT';
+
+    private SoapClient $client;
```

Comment:
```suggestion
    protected SoapClient $client;
```

Reply:
Why would we make this protected when the class is final?